http/codegen/service_data.go

```diff
@@ -769,8 +769,12 @@ func (d ServicesData) analyze(hs *expr.HTTPServiceExpr) *ServiceData {
 			)
 			{
 				name = fmt.Sprintf("Build%sRequest", ep.VarName)
+				s := codegen.NewNameScope()
+				s.Unique("c") // 'c' is reserved as the client's receiver name.
 				for _, ca := range routes[0].PathInit.ClientArgs {
 					if ca.FieldName != "" {
+						ca.Name = s.Unique(ca.Name, "_")
```

Comment:
Instead of using `Unique` you can use `Name` which will automatically suffix a counter variable to the name. You don't have to suffix with `_`
```
ca.Name = s.Name(ca.Name)
```